### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wangjingz**
Add Status badge for openSSF, Dependency, CodeQL and License

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Enhanced BasicVSR (iVSR)
 
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/OpenVisualCloud/Video-Super-Resolution-Library.Enhanced-BasicVSR/badge)](https://api.securityscorecards.dev/projects/github.com/OpenVisualCloud/Video-Super-Resolution-Library.Enhanced-BasicVSR)
+[![Dependency Review](https://github.com/OpenVisualCloud/Video-Super-Resolution-Library.Enhanced-BasicVSR/actions/workflows/dependency-review.yml/badge.svg)](https://github.com/OpenVisualCloud/Video-Super-Resolution-Library.Enhanced-BasicVSR/actions/workflows/dependency-review.yml)
+[![CodeQL](https://github.com/OpenVisualCloud/Video-Super-Resolution-Library.Enhanced-BasicVSR/actions/workflows/codeql.yml/badge.svg)](https://github.com/OpenVisualCloud/Video-Super-Resolution-Library.Enhanced-BasicVSR/actions/workflows/codeql.yml)
```
Comment (djie1) [inline README.md:5]: Will we merge PR#13 first?

**Review by djie1 — DISMISSED** [review]
LGTM

**Comment by djie1** [comment]
Suppose we will merge PR#11 first, and then rebase this PR.

**Review by wangjingz — COMMENTED** [review]
Add badge display in GitHub repo.

**Pull request closed without merging**